contentreview-impl/tool/pom.xml

```diff
@@ -41,6 +41,10 @@
             <groupId>org.sakaiproject.contentreview</groupId>
             <artifactId>contentreview-service-api</artifactId>
         </dependency>
+		<dependency>
```

Comment:
I know, like I said elsewhere didn't want to include a such specific TII LTI method (obtainGlobalToolData) on the CR-API. Like this tool only has the servlets I added doesn't seem like a problem.